spec/factories/sender_infos.rb

```diff
@@ -0,0 +1,23 @@
+FactoryGirl.define do
+  factory :sender_info do
```

Comment:
Uznałem, że de facto są to dwa osobne byty.
Nie chciałbym, żeby zmiany w jednym modelu propagowały się na drugi, chociażby - obecnie są identyczne, ale to nie musi tak pozostać.
Rozwiązanie, w którym szukam powiązań między nimi może dużo zagmatwać dla niedużej korzyści.